### PR TITLE
[SettingsBundle] Settings namespaces permissions

### PR DESCRIPTION
**Pull request by gperdomor**
Hi...

This simple PR validate the permission of settings namespace inside SettingsController.

### CODE REVIEW

File: src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php [hunk 1]
```diff
@@ -49,20 +51,21 @@ public function showAction(Request $request)
 
         $view = $this
             ->view()
-            ->setData($settings)
-        ;
+            ->setData($settings);
```
Comment (tristanbes) [inline src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php]: the `;` should be on a new line like it was before. Makes cleaner diff.

File: src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php [hunk 2]
```diff
@@ -77,8 +80,7 @@ public function updateAction(Request $request, $namespace)
 
         $form = $this
             ->getSettingsFormFactory()
-            ->create($namespace, $settings, $isApiRequest ? array('csrf_protection' => false) : array())
-        ;
+            ->create($namespace, $settings, $isApiRequest ? array('csrf_protection' => false) : array());
```
Comment (tristanbes) [inline src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php]: it should be reverted too

**Comment by gperdomor** [comment]
@tristanbes done... Also run the <code>sylius-cs-fixer</code> tool to fix some CS

**Comment by tristanbes** [comment]
You should squash your commit into 1, this looks messy for little code manipulated. Thanks :-)

**Comment by pjedrzejewski** [comment]
@gperdomor Perhaps we should add these permissions to sylius.yml?

**Comment by gperdomor** [comment]
yes, i forgot update the permissions. My bad. Updated now

**Comment by pjedrzejewski** [comment]
Thanks @gperdomor! :+1:

**Pull request merged**